editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy-multi-file.spec.browser2.tsx

```diff
@@ -309,7 +309,7 @@ export var App = () => {
       <FlexParent data-uid='flexparentinapp' />
       <AbsoluteChild
         data-uid='absolutechildinparent'
-        style={{ left: 294, top: -40 }}
+        style={{ left: 294, top: -40, position: 'absolute' }}
```

Comment:
This shouldn't be needed, because the component already has `position: 'absolute'` on its root element.